swagger/body-time.json

```diff
@@ -0,0 +1,86 @@
+{
+    "swagger": "2.0",
+    "info": {
+      "title": "AutoRest Time Test Service",
+      "description": "Test Infrastructure for AutoRest",
+      "version": "1.0.0"
+    },
+    "host": "localhost:3000",
+    "schemes": [
+      "http"
+    ],
+    "produces": [
+      "application/json"
+    ],
+    "consumes": [
+      "application/json"
+    ],
+    "paths": {
+      "/time/get": {
+        "get": {
+          "operationId": "time_get",
+          "description": "Get time value",
+          "responses": {
+            "200": {
+              "description": "The time value",
```

Comment:
Can you please include the returned value in description, it's easier to write test that way.